packages/oui-tile/README.md

```diff
@@ -0,0 +1,71 @@
+# oui-tile
+
+<component-status cx-design="partial" ux="prototype"></component-status>
+
+oui-tile is a package which provides styles for the `oui-tile` component.
+
+## Installation
+
+```less
+@import 'oui-tile/tile';
+```
+
+## Usage
+
+### Buttons
+
+```html:preview
+<div class="oui-tile">
+  <h4 class="oui-header_4 oui-tile__title">Buttons</h4>
+  <div class="oui-tile__body">
+    <div class="oui-tile__item oui-tile__item_button">
+      <button class="oui-tile__button oui-button oui-button_link oui-button_icon-right oui-button_full-width">
```

Comment:
It could be better if you use on your `button` element this following attribute: `type="button"`